src/citrine/seeding/dataset_deletes.py

```diff
@@ -0,0 +1,53 @@
+"""Functions that perform dataset deletes."""
+from typing import List, Tuple
+
+from citrine.resources.api_error import ApiError
+from citrine.resources.dataset import Dataset
+
+from gemd.entity.link_by_uid import LinkByUID
+
+
+def wipe_dataset(dataset: Dataset, delete_templates: bool = False) -> \
```

Comment:
```suggestion
def wipe_dataset(dataset: Dataset, *, delete_templates: bool = False) -> \
```
Require the key when using that argument.